docker/lib/dependabot/docker/file_fetcher.rb

```diff
@@ -9,6 +9,7 @@ module Docker
     class FileFetcher < Dependabot::FileFetchers::Base
       YAML_REGEXP = /^[^\.]+\.ya?ml$/i.freeze
       DOCKER_REGEXP = /dockerfile/i.freeze
+      HELM_REGEXP = /values.yaml/i.freeze
```

Comment:
Minor nitpick: `/values\.yaml/i`

Reply:
good catch. Fixed and added a lot of unit tests for this regexp.